query-service/src/integrationTest/resources/sql/functions.sql

```diff
@@ -1 +1,31 @@
-CREATE OR REPLACE FUNCTION dateTimeConvert (bigint, bigint) RETURNS bigint AS $$ select ((($1 + $2 - 1)/$2)*$2) $$ LANGUAGE SQL;
\ No newline at end of file
+CREATE OR REPLACE FUNCTION dateTimeConvert (bigint, bigint) RETURNS bigint AS $$ select ((($1 + $2 - 1)/$2)*$2) $$ LANGUAGE SQL;
+
+CREATE OR REPLACE FUNCTION conditional (condition text, first text, second text)
```

Comment:
should condition be text or boolean? similarly, is there any kind of "any" type for first and second?

Reply:
That is what we discussed yesterday right that condition can be null as well. That is reason I have used text, and converting to boolean only if it is not null.

All arguments must be of specific type in postgres functions.